.github/workflows/release_candidate.yml

```diff
@@ -59,3 +60,57 @@ jobs:
                 target_url: `https://unpkg.com/${package.name}@${package.version}/`
               })
             })
+
+  release-candidate-next-minor:
+    if: ${{ github.repository == 'primer/brand' && github.ref_name == 'changeset-release/next-minor' }}
+    name: Next minor candidate (@next-minor)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build:lib
+
+      - name: Create .npmrc
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN_SHARED }}
+
+      - name: Publish release candidate
+        run: |
+          npm exec --workspaces -- ../../packages/repo-configs/scripts/prepare-release-candidate
+          yarn changeset publish --tag next-minor
```

Comment:
FYI: Everything here is the same, except for this part where I'm setting the release channel to be next-minor ☝️